lib/Options.php

```diff
@@ -206,6 +206,39 @@ public static function should_run_unbabble() : bool {
 		return ! in_array( get_current_blog_id(), UNBABBLE_IDLE, true );
 	}
 
+	/**
+	 * Get Unbabble options for a specific blog in a multisite installation.
+	 *
+	 * @param  int $blog_id
+	 * @return ?array
+	 */
+	public static function get_blog_options( int $blog_id ) : ?array {
```

Comment:
I think this should be inside de logic for the `get()` function. The `$site_id` would be an optional parameter, maybe inside an array for future proofing, and then the switching any related filter would carry the ID value, if needed.